asab/config.py

```diff
@@ -206,6 +217,26 @@ def _load(self):
 
 
 	def _include_from_zookeeper(self, zkurl):
+		"""
```

Comment:
it can read the zkurl from environmental variable as well

Reply:
@mejroslav you can add this to doctoring too.